File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Convert Markdown to Word document in C# | DocIO | Syncfusion
+description: Convert Markdown to Word document in C# using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Markdown to Word Conversion
+
+Markdown is a lightweight markup language that adds formatting elements to plain text documents. The .NET Word (DocIO) library supports to convert Markdown to Word document and vice versa, which mostly follows the CommonMark specification and GitHub-flavored syntax.
```

Comment:
Markdown is a lightweight markup language that adds formatting elements to plain text documents. The .NET Word (DocIO) library supports converting Markdown to Word document and vice versa, which mostly follows the CommonMark specification and GitHub-flavored syntax.